tests/functional/products/test_products.py

```diff
@@ -139,8 +139,9 @@ def test_get_product_data(self):
                          "The product's endpoint is improper.")
         self.assertTrue(pr_data.id > 0, "Product didn't have valid ID")
 
-        self.assertTrue(pr_data.connected, "Products should be connectible "
-                        "in a test environment.")
+        self.assertFalse(pr_data.connected,
+                         "Deprecated value will take always False."
```

Comment:
Will these tests be updated later on? **We are losing a dimension here** to see if calling `editProduct` actually makes the server use the **new** (edited) connection information under the same product endpoint.

At least introduce a TODO in this test to explain this!

Reply:
I've added comment to the other test change too before. That the schema will not be automatically created in the new database after editing a product configuration.